### PR TITLE
Adding unbundling attempts in HTTP/2

### PR DESCRIPTION
**Pull request by ylafon**
Reference to http/3 and quick as well.
This addresses issue #5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -131,16 +131,22 @@
 
 ## The Hidden Cost of Bundling
 
-Bundling lowers latency and gives coherence to the whole package,
+Bundling lowers latency, lowers the protocol cost by reducing the number of
```
Comment (LeaVerou) [inline index.html]: Not sure what "protocol cost" is?

File: index.html [hunk 2]
```diff
@@ -131,16 +131,22 @@
...
+requests and gives coherence to the whole package,
```
Comment (LeaVerou) [inline index.html]: What does "gives coherence to the whole package" actually mean? (I know it was there already but still)

File: index.html [hunk 3]
```diff
@@ -131,16 +131,22 @@
...
 but the hidden cost is the unused payload
 and the recurring cost of it during an update.
 There is also the inherent risk of “write amplification”,
 where even a single byte change in a web application requires
-downloading the entire application from scratch.
+downloading the entire application from scratch, which is the extreme case of the partial update issue.
```
Comment (LeaVerou) [inline index.html]: As a reader, I'm not sure what this means.

File: index.html [hunk 4]
```diff
@@ -131,16 +131,22 @@
...
 There is a definite cost associated with this,
 both from the end of providing the application,
 as well as receiving the application.
 
+There have been attempts to unbundle, especially libraries when HTTP/2 [[rfc9113]]
+was introduced, using Server Push, but there was no noticeable enough performance gain
+to alleviate the complexity cost.
+The introduction of HTTP/3 [[rfc9114]], based on QUIC [[rfc9100]] expects to improve this again.
```
Comment (LeaVerou) [inline index.html]:
```suggestion
The introduction of HTTP/3 [[rfc9114]], based on QUIC [[rfc9100]] may provide new solutions to these issues.
```

(It doesn't quite make sense that "the introduction" of something "expects" anything)

File: index.html [hunk 5]
```diff
@@ -131,16 +131,22 @@
...
 where even a single byte change in a web application requires
-downloading the entire application from scratch.
+downloading the entire application from scratch, which is the extreme case of the partial update issue.
 There is a definite cost associated with this,
 both from the end of providing the application,
 as well as receiving the application.
 
+There have been attempts to unbundle, especially libraries when HTTP/2 [[rfc9113]]
+was introduced, using Server Push, but there was no noticeable enough performance gain
+to alleviate the complexity cost.
```
Comment (LeaVerou) [inline index.html]:
```suggestion
When HTTP/2 [[rfc9113]] was introduced, attempts were made to alleviate the need for bundling using Server Push, but the performance gain was not noticeable enough to justify the complexity cost.
```

(Not 100% sure I understood the meaning correctly, but I found the original sentence quite confusing)

**Review by LeaVerou — CHANGES_REQUESTED** [review]
The language reads a bit confusing to me in several places, see comments

File: index.html (hunk 2 above)
Comment (torgo) [reply index.html]:
```suggestion
requests and makes the package atomic,
```

File: index.html (hunk 3 above)
Comment (torgo) [reply index.html]:
```suggestion
downloading the entire application from scratch.
```

File: index.html (hunk 1 above)
Comment (torgo) [reply index.html:134]:
```suggestion
Bundling lowers latency, lowers the protocol chattiness by reducing the number of
```

Comment (cynthia) [reply index.html]:
```suggestion
Bundling lowers latency, lowers the transport cost by reducing the number of
```

File: index.html (hunk 5 above)
Comment (cynthia) [reply index.html]: Nit: Line breaks as well.

**Review by cynthia — COMMENTED** [review]
Minor editorial nits, but otherwise LGTM.

**Pull request merged**